nuspec/netcoreglobaltool/dotnet-sonarscanner.nuspec

```diff
@@ -21,14 +21,14 @@ All release notes for SonarScanner .Net Core can be found on the GitHub site - h
     </packageTypes>
     <dependencies>
       <group targetFramework=".NETCoreApp2.1" />
-      <group targetFramework=".NETCoreApp3.0" />
+      <group targetFramework=".NETCoreApp3.1" />
     </dependencies>
   </metadata>
   <files>
-    <file src="..\..\DeploymentArtifacts\BuildAgentPayload\$configuration$\netcoreapp2.1\" target="tools\netcoreapp2.1\any" />
-    <file src="..\..\DeploymentArtifacts\BuildAgentPayload\$configuration$\netcoreapp3.0\" target="tools\netcoreapp3.0\any" />
+    <file src="..\..\build\sonarscanner-msbuild-netcoreapp2.0\" target="tools\netcoreapp2.1\any" />
```

Comment:
I'm confused - why is the nuspec referencing 2.0 and 3.0? Isn't it only building 2.1 and 3.1 now?

Reply:
This is just a matter of folder's naming, i keep 2.0/3.0 for consistency on the CI, but you're right, we're in 2.1/3.1 now.